sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -214,6 +218,34 @@ export namespace ConnectionContext {
           return waitForDisconnectPromise;
         }
         return Promise.resolve();
+      },
+      async close() {
```

Comment:
Nice, this feels like one of those things that should have always been here :smile: